src/components/RecentEventsCard.astro

```diff
@@ -9,7 +23,7 @@ const { title, pictureUrl, date, description } = event;
 >
   <div class="w-full h-4/6 overflow-hidden">
     <img
```

Comment:
1. Please turn this into a div with a background image and fixed height instead. We need to use  `bg-cover`.
    - See sample implementation here: https://github.com/SPARCS-UP-Mindanao/TechTix/blob/ced512e6b28701ea0306eba4a53c308e16c6039c/frontend/src/components/EventCard/EventCard.tsx#L28
2. Use the new sparcs logo icon as a placeholder if image link fetching has failed.
    - See sample implementation here: https://stackoverflow.com/questions/37588017/fallback-background-image-if-default-doesnt-exist